modules/local/report.nf

```diff
@@ -30,87 +30,86 @@ process REPORT {
 
     script:
 
-    def contrast_matrix_opt = contrast_matrix.name != 'DEFAULT' ? "--contrast_matrix $contrast_matrix" : ''
-    def contrast_list_opt = contrast_list.name != 'DEFAULT1' ? "--contrast_list $contrast_list" : ''
-    def contrast_pairs_opt = contrast_pairs.name != 'DEFAULT2' ? "--contrast_pairs $contrast_pairs" : ''
-    def genelist_opt = genelist.name != 'NO_FILE' ? "--genelist $genelist" : ''
-    def relevel_opt = relevel.name != 'NO_FILE2' ? "--relevel $relevel" : ''
+    def contrast_matrix_opt = contrast_matrix.name != 'DEFAULT' ? "--contrast_matrix ${contrast_matrix}" : ''
```

Comment:
Just a comment here: Alll of the `${}` were automatically reformatted upon save.

Reply:
I think thats fine, but makes it a little less consistent ;)